### PR TITLE
[Task #50] Create "meal type" section of the default page view 

### PR DESCRIPTION
**Pull request by piotrekg35**

### CODE REVIEW

**Review by Aleksander2a — CHANGES_REQUESTED** [review]
I think the pictures could be a bit higher (bigger) and the text should also have a different font style (in my opinion  sans-serif matches the figma design best), color (white)  and I'd make it bold and bigger just a bit, but it's up to you

File: meal_type/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section id="meal-types">
+        <div class="content">
+            <div class="meal">
+                <h3>Breakfast</h3>
```
Comment (Laronk) [inline meal_type/index.html:13]: The text contained in h3 html tags should have css style text-transform set to uppercase.

File: meal_type/style.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+html, body{
```
Comment (Laronk) [inline meal_type/style.css:1]: reformat file with css linter (stylelint in vscode)

File: meal_type/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    padding: 0;
+    margin: 0;
+}
+#meal-types{
+    background-color: #E5E5E5;
+    
+
+}
+.meal{
```
Comment (Laronk) [inline meal_type/style.css:10]: Each CSS selector should be preceded with '#meal-types' - section id. This way after merging all the different sections together the formatting for each specific section will be applied accordingly.

File: meal_type/style.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    float: left;
+    width: 33.33%;
+
+}
+
+h3{
```
Comment (Laronk) [inline meal_type/style.css:16]: remove bottom padding from h3 tag

**Review by Laronk — CHANGES_REQUESTED** [review]

File: meal_type/index.html (hunk 1 above)
Comment (piotrekg35) [reply meal_type/index.html:13]: Fixed. See 340cd012b81d32dae423998f88c8908cde57163d

File: meal_type/style.css (hunk 2 above)
Comment (piotrekg35) [reply meal_type/style.css:1]: I had a problem with this one. I have downloaded a linter, but it doesn't work.

File: meal_type/style.css (hunk 3 above)
Comment (piotrekg35) [reply meal_type/style.css]: Fixed. See 340cd012b81d32dae423998f88c8908cde57163d

File: meal_type/style.css (hunk 4 above)
Comment (piotrekg35) [reply meal_type/style.css]: Fixed. See 340cd012b81d32dae423998f88c8908cde57163d

Comment (Laronk) [reply meal_type/style.css]: @piotrekg35 I meant to  write margin, sorry

File: meal_type/style.css (hunk 2 above)
Comment (Laronk) [reply meal_type/style.css:1]: Try this extention instead https://marketplace.visualstudio.com/items?itemName=aeschli.vscode-css-formatter

Comment (piotrekg35) [reply meal_type/style.css]: Fixed. See 84d7be84559b28d34cea15c0ae3da8742451703b

**Review by Aleksander2a — APPROVED** [review]
I think it's good

**Review by Laronk — APPROVED** [review]
Ok

**Pull request merged**